docs/content/en/concepts/quickstart-tenancy-and-apis.md

```diff
@@ -263,25 +289,61 @@ metadata:
 status:
   ...
   identityHash: a6a0cc778bec8c4b844e6326965fbb740b6a9590963578afe07276e6a0d41e20
-  virtualWorkspaces:
-  - url: https://myhost:6443/services/apiexport/root:wildwest:cowboys-service/wildwest.dev
 ```
 
-We can see that our `APIExport` has two key attributes in its status - its identity (more on this below) and a "virtual
-workspace" URL. You can think of this virtual workspace as behaving just like a workspace or cluster, except it searches
-across all true workspaces for instances of the resource types provided by the `APIExport`. We can use API discovery to
-see what resource types are available via this virtual workspace:
+We can see that our `APIExport` has a key attribute in its status - its identity (more on this below).
+This identity can be used in permissionClaims for referring to non-core resources.
+
+## APIExportEndpointSlice
+
+`APIExportEndpointSlices` allow service provider to retrieve the URL of service endpoints, acting as a sink for them. You can think of this endpoint as behaving just like a workspace or cluster, except it searches across all workspaces for instances of the resource types provided by the `APIExport`.
+An `APIExportEndpointSlice` is created by a service provider, references a single `APIExport` and optionally a `Partition`.
+`Partitions` are a mechanism for filtering service endpoints. Within a multi-sharded kcp, each shard will offer its own service endpoint URL for an `APIExport`. Service provider may decide to have multiple instances of their controller reconciliating, for instance, resources of shards in the same region. For that they may create an `APIExportEndpointSlice` in the same workspace where a controller instance is deployed. This `APIExportEndpointSlice` will then reference a specific `Partition`in the same workspace filtering the service endpoints for a subset of shards. If an `APIExportEndpointSlice` does not reference a `Partition` all the available endpoints are populated in its `status`. More on `Partitions` [here](./partitions,md).
+
+```shell
+$ kubectl apply -f - <<EOF
+kind: APIExportEndpointSlice
+apiVersion: apis.kcp.io/v1alpha1
+metadata:
+    name: cowboys
+spec:
+    export:
+        path: root:wildwest:cowboys-service
+        name: cowboy
+    # optional
+    partition:  us-west-1
```

Comment:
```suggestion
    partition:  cloud-region-gcp-europe-xdfgs
```

Is this by name? Could we reference one of the example above?

Reply:
Could one reference a PartitionSet instead?

Reply:
No, only Partitions can be referenced. PartitionSets are just a convenient way of generating Partitions. If service providers know the Shards and their labels they could create Partitions directly without using a PartitionSet.
I can change the name.